credentialAtomicQueryMTPWithRelay.go

```diff
@@ -0,0 +1,223 @@
+package circuits
+
+import (
+	"errors"
+	core "github.com/iden3/go-iden3-core"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+	"github.com/iden3/go-merkletree-sql"
+	"math/big"
+	"strconv"
+)
+
+const (
+	// AtomicQueryMTPWithRelayPublicSignalsSchema is schema to parse json data for additional information
+	AtomicQueryMTPWithRelayPublicSignalsSchema PublicSchemaJSON = `{"userID":0, "relayState":1,"challenge":2,"claimSchema":3,"slotIndex":4,"operator":5,"value":6,"timestamp":7, "issuerID":8}`
```

Comment:
can you change userID to "user_identifier" This will be similar to other circuits.and auth lib will work.
In the next PR we will fix naming in all circuits.